### PR TITLE
OMEdit: Use splitCommand for terminal args

### PR DESCRIPTION
**Pull request by jschueller**
Use `QProcess::splitCommand` instead of `QString::split(" ")`, which allows to use quotes, as seen in #9606

cc @adeas31

### CODE REVIEW

File: OMEdit/OMEditLIB/Util/Utilities.h [hunk 1]
```diff
@@ -417,6 +417,8 @@ class QDetachableProcess : public QProcess
   Q_OBJECT
 public:
   QDetachableProcess(QObject *pParent = 0);
+
+  using QProcess::start;
```
Comment (adeas31) [inline OMEdit/OMEditLIB/Util/Utilities.h:421]: Is this necessary?

**Review by adeas31 — APPROVED** [review]
LGTM.

File: OMEdit/OMEditLIB/Util/Utilities.h (hunk 1 above)
Comment (jschueller) [reply OMEdit/OMEditLIB/Util/Utilities.h:421]: yes, else we cannot call the start(QString) method overload

**Comment by adeas31** [comment]
> side question, no nightly builds today ? https://build.openmodelica.org/omc/builds/windows/nightly-builds/

Yeah sorry about that. I will fix them asap.

File: OMEdit/OMEditLIB/Util/Utilities.h (hunk 1 above)
Comment (perost) [reply OMEdit/OMEditLIB/Util/Utilities.h:421]: `using` pulls in all the overloads.

Comment (jschueller) [reply OMEdit/OMEditLIB/Util/Utilities.h]: @adeas31 good point

**Pull request merged**